tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1777,34 +1828,35 @@ schedule(void)
             i->flags |= LOOP_BACKEDGE_FLAG;
             i->misc_metadata = loop_md;
           }
-        }
-        if (BIH_UNROLL(bih)) {
-          LL_MDRef loop_md = cons_unroll_metadata();
-          INSTR_LIST *i = find_last_executable(llvm_info.last_instr);
-          if (i) {
-            i->flags |= LOOP_BACKEDGE_FLAG;
-            i->misc_metadata = loop_md;
-          }
+	}
```

Comment:
Incorrect indentation. @michalpasztamobica  Could you also fix this one?

Reply:
Done :)